doc/install.rst

```diff
@@ -28,15 +28,19 @@ or ``conda``::
 
     conda install scikit-learn
 
-**We don't recommend installing scipy or numpy using pip on linux**,
-as this will involve a lengthy build-process with many dependencies.
-Without careful configuration, building numpy yourself can lead to an installation
-that is much slower than it should be. 
+**We don't recommend installing scipy or numpy using pip on Linux**,
+as this will involve a lengthy build process with many dependencies.
+Without careful configuration, building numpy yourself is likely to lead to an
+installation that is much slower than it should be.
 If you are using Linux, consider using your package manager to install
 scikit-learn. It is usually the easiest way, but might not provide the newest
 version.
 If you haven't already installed numpy and scipy and can't install them via
-your operation system, it is recommended to use a third party distribution.
+your operating system, we recommended using a third-party distribution.
+
+If you must install scikit-learn with pip, you can install it as
```

Comment:
Clarify that then?

On 23 August 2016 at 12:55, Jimmy Jia <notifications@github.com> wrote:

> In doc/install.rst
> <https://github.com/scikit-learn/scikit-learn/pull/6990#discussion_r75795830>
> :
>
> >  If you are using Linux, consider using your package manager to install
> >  scikit-learn. It is usually the easiest way, but might not provide the newest
> >  version.
> >  If you haven't already installed numpy and scipy and can't install them via
> > -your operation system, it is recommended to use a third party distribution.
> > +your operating system, we recommended using a third-party distribution.
> > +
> > +If you must install scikit-learn with pip, you can install it as
>
> I'll go with "and its dependencies". The target use case isn't typing pip
> install scikit-learn[alldeps] on the command line – it's instead having a
> single requirements.txt that lists everything.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/6990/files/83d14b234d3be76737a15f12f8dc99696d0bf4d3#r75795830>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEz64iSp68aVAfdWDAGSObo9_PV2MIzks5qimEtgaJpZM4JM0Pv>
> .
>